### PR TITLE
Java 11 migration: ambassador async-method-invocation balking bridge builder

### PR DESCRIPTION
**Pull request by anuragagarwal561994**
Migrates:
ambassador
async-method-invocation
balking
bridge
builder

### CODE REVIEW

File: ambassador/README.md [hunk 1]
```diff
@@ -146,10 +146,14 @@ public class Client {
...
 
 ```java
-Client host1 = new Client();
-Client host2 = new Client();
-host1.useService(12);
-host2.useService(73);
+public class App {
+  public static void main(String[] args) {
+    Client host1 = new Client();
+    Client host2 = new Client();
```
[the remark this answers is not in the document]
Comment (anuragagarwal561994) [reply ambassador/README.md]: We can and I was about to change this as well, but IDEs ans formatters currently doesn't understand var. They won't format and thus I didn't push that change. Also most of the people are currently on java8 so if anyone is trying and testing from the docs won't it be good to let it be in java8 format?

[the remark this answers is not in the document]
Comment (iluwatar) [reply ambassador/README.md]: We are changing all patterns to use `var` syntax. If not modified here, then we will do it in issue https://github.com/iluwatar/java-design-patterns/issues/984

**Comment by iluwatar** [comment]
Updated readmes in commits https://github.com/iluwatar/java-design-patterns/commit/329479dda0d7bc0bafb0c8101d1cf710a8739b44, https://github.com/iluwatar/java-design-patterns/commit/0272d718514ba718ca97b4562db843345d7c077f and https://github.com/iluwatar/java-design-patterns/commit/2628cc0dfc82108ae862452818ddd911d5682ebd. Now we can mark https://github.com/iluwatar/java-design-patterns/issues/984 and https://github.com/iluwatar/java-design-patterns/issues/987 completed for these patterns.

**Comment by iluwatar** [comment]
Thanks @anuragagarwal561994 for yet another contribution. I really appreciate your efforts :relaxed:

**Pull request merged**